staging/src/k8s.io/component-base/logs/logs.go

```diff
@@ -33,12 +33,21 @@ var logFlushFreq = pflag.Duration(logFlushFreqFlagName, 5*time.Second, "Maximum
 
 func init() {
 	klog.InitFlags(flag.CommandLine)
+	initLogRegistry()
```

Comment:
Can you call `initLogRegistry` in `registry.go` and rename it to `init` ?
I don't see benefit of having one `init` per package as it's not obvious from 'logs.go' package that it requires registry.